shared/common-adapters/markdown/react.js

```diff
@@ -328,6 +327,23 @@ const reactComponentsForMarkdownType = {
       </React.Fragment>
     )
   },
+  mailto: (node, output, state) => {
+    return (
+      <React.Fragment key={state.key}>
+        {node.spaceInFront}
+        <Text
+          className="hover-underline"
+          type="BodyPrimaryLink"
+          style={Styles.collapseStyles([linkStyle, state.styleOverride?.mailto])}
```

Comment:
dont need the ?. right

Reply:
whoops a remnat left over from the before times!
```suggestion
          style={Styles.collapseStyles([linkStyle, state.styleOverride.mailto])}
```